### PR TITLE
add a case for dubbo-samples-gateway-apisix-dubbo

### PR DESCRIPTION
**Pull request by mming404**
add a case for dubbo-samples-gateway-apisix-dubbo

### CODE REVIEW

**Comment by mming404** [comment]
PTAL @chickenlj

File: 2-advanced/dubbo-samples-gateway/dubbo-samples-gateway-apisix/dubbo-samples-gateway-apisix-dubbo/README.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+- DEB
+- Source Code
+
+This document uses Docker installation, which is relatively simple
+
+To install APISIX using this method, you need to install Docker（ [https://www.docker.com/](https://www.docker.com/) ）And Docker Composite（ [https://docs.docker.com/compose/](https://docs.docker.com/compose/) ）.
+
+First, download [apisix Docker]（ [https://github.com/apache/apisix-docker](https://github.com/apache/apisix-docker) ）Warehouse.
+
+`git clone https://github.com/apache/apisix-docker.gitcd apisix-docker/example`
```
Comment (chickenlj) [inline 2-advanced/dubbo-samples-gateway/dubbo-samples-gateway-apisix/dubbo-samples-gateway-apisix-dubbo/README.md:19]: Please split these two commands.

File: 2-advanced/dubbo-samples-gateway/dubbo-samples-gateway-apisix/dubbo-samples-gateway-apisix-dubbo/README.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+- Helm
+- RPM
+- DEB
+- Source Code
+
+This document uses Docker installation, which is relatively simple
+
+To install APISIX using this method, you need to install Docker（ [https://www.docker.com/](https://www.docker.com/) ）And Docker Composite（ [https://docs.docker.com/compose/](https://docs.docker.com/compose/) ）.
+
+First, download [apisix Docker]（ [https://github.com/apache/apisix-docker](https://github.com/apache/apisix-docker) ）Warehouse.
```
Comment (chickenlj) [inline 2-advanced/dubbo-samples-gateway/dubbo-samples-gateway-apisix/dubbo-samples-gateway-apisix-dubbo/README.md:17]: replace ` [apisix Docker]（ [https://github.com/apache/apisix-docker](https://github.com/apache/apisix-docker) ）` with ` [apisix-docker](https://github.com/apache/apisix-docker) `

Comment (chickenlj) [reply 2-advanced/dubbo-samples-gateway/dubbo-samples-gateway-apisix/dubbo-samples-gateway-apisix-dubbo/README.md:17]: Also replace similar contents above

File: 2-advanced/dubbo-samples-gateway/dubbo-samples-gateway-apisix/dubbo-samples-gateway-apisix-dubbo/dubbo-samples-gateway-apisix-dubbo-interface/src/main/java/org/apache/dubbo/samples/api/DemoService.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.samples.api;
+
+/**
+ * @Description: TODO
+ * @Author MiSinG
```
Comment (chickenlj) [inline 2-advanced/dubbo-samples-gateway/dubbo-samples-gateway-apisix/dubbo-samples-gateway-apisix-dubbo/dubbo-samples-gateway-apisix-dubbo-interface/src/main/java/org/apache/dubbo/samples/api/DemoService.java:21]: Please remove @author information.

**Review by chickenlj — CHANGES_REQUESTED** [review]

**Pull request merged**